contracts/contracts/sbtc-bootstrap-signers.clar

```diff
@@ -1,17 +1,14 @@
-
-;; title: sbtc-bootstrap-signers
-;; version:
-;; summary:
-;; description:
-
-;; traits
-;;
-
-;; token definitions
-;;
+;; sBTC Bootstrap Signers contract
 
 ;; constants
-;;
+(define-constant signature-threshold u8)
+(define-constant key-size u32)
+
+;; errors
+;; if err is u200, it's the agg key
+;; if err is u210>, it's the key at index (err - 210)
```

Comment:
This is probably fine, but we'd likely want to refactor if this contract ends up having more than 10 error codes

Reply:
Copy, good point out. Will leave as-is but will change to "220"+ if we do have more than 10.